### ISSUE TITLE
no default directory, certutil not found

### ISSUE THREAD

**Issue by sneakyimp** [bagder/curl#207]
I tried running firefox-db2pem.sh on an Ubuntu machine and got this error:
```
$ ./firefox-db2pem.sh
ls: cannot access /home/sneakyimp/.mozilla/firefox/*default: No such file or directory
./firefox-db2pem.sh: 46: ./firefox-db2pem.sh: certutil: not found
```

**Comment by bagder** [comment]
In debian certutil comes in the libnss3-tools package. The *default dir is what the script assumes your Firefox data should be in. Where's yours?

**Comment by sneakyimp** [comment]
Thanks for the tip RE: certutils.

I'm not sure exactly which folder you are looking for, but it looks like the firefox 'default' dir has its name altered to add some randomized bits both before and after the word default. I've changed some of the chars here but you should get the idea:
```
$ ls -al ~/.mozilla/firefox
total 20
drwx------  4 sneakyimp sneakyimp 4096 Aug 22  2014 .
drwx------  4 sneakyimp sneakyimp 4096 Oct 25  2013 ..
drwx------  5 sneakyimp sneakyimp 4096 Apr  6 17:37 Crash Reports
-rw-rw-r--  1 sneakyimp sneakyimp  132 Aug 22  2014 profiles.ini
drwx------ 16 sneakyimp sneakyimp 4096 Apr  7 14:29 uXXXXXXX.default-1234567890123
```
so that would be
```
/home/sneakyimp/.mozilla/firefox/uXXXXXXX.default-1234567890123
```
And inside that directory there's a file called cert8.db which appears to contain the CA bundle.


I have installed libnss3-tools and altered this line:
```
db=`ls -1d $HOME/.mozilla/firefox/*default`
```
to this:
```
db=`ls -1d $HOME/.mozilla/firefox/*default*`
```
And it seems to run, generating a ca-bundle.crt file which looks to be PEM encoded.

**Comment by bagder** [comment]
Thanks, fixed in 7fe172d3b2cc0
